lib/windows.js

```diff
@@ -6,6 +6,17 @@ var request = require('request');
 var url = 'https://lic.apps.microsoft.com/licensing/certificateserver/?cid=';
 var sigXPath = '//*//*[local-name(.)=\'Signature\' and namespace-uri(.)=\'http://www.w3.org/2000/09/xmldsig#\']';
 
+module.exports.readConfig = function (configIn) {
+	if (!configIn) {
+	    // no required config
+	    return;
+	}
+	// Apply any default settings to Request.
+	if ('requestDefaults' in configIn) {
+		request = request.defaults(configIn['requestDefaults']);
```

Comment:
My bad... I should have caught this the first time around, but would change this line to use dot-notation instead?

request = request.defaults(configIn.requestDefaults);

Sorry. I'm a little picky ;-)